firebase-functions/functions/src/operations/playerOperations.ts

```diff
@@ -1,32 +1,57 @@
-﻿import { getPlayersToDeactivate } from './choiceOperations';
-import * as admin from 'firebase-admin';
+﻿import * as admin from 'firebase-admin';
 import { firestore } from 'firebase-admin';
 import { collections } from '../constants/collections';
 import { Player } from '../models/player';
 import DocumentData = firestore.DocumentData;
 import QueryDocumentSnapshot = firestore.QueryDocumentSnapshot;
+import { Game } from '../models/game';
+import { GameType } from '../models/game-type';
+import { Result } from '../models/result';
+import { getCurrentGameType } from './roomOperations';
+import { addLastOneActiveToGame } from './GameOperations';
 
-export async function deactivatePlayers(roomId: string, initiallyActivePlayers: QueryDocumentSnapshot<DocumentData>[]): Promise<void> {
-  const playersToDeactivate = await getPlayersToDeactivate(initiallyActivePlayers, roomId);
+export async function deactivatePlayers(roomId: string, game: Game, initiallyActivePlayers: QueryDocumentSnapshot<DocumentData>[]): Promise<void> {
+  const playersToDeactivate = await getPlayersToDeactivate(game, roomId);
```

Comment:
Maybe rename this to `playerIdsToDeactivate` (same in method). Little more precise?